### PR TITLE
[4474] Use material design icons all across UI next and remove font awesome

### PR DESCRIPTION
**Pull request by jvega190**
https://github.com/craftercms/craftercms/issues/4474

### CODE REVIEW

**Comment by jvega190** [comment]
Blueprints configs need to be updated, waiting for approval of ticket in case there are config updates pending

File: ui/app/src/assets/uiConfigDefaults.ts
Comment (rart) [inline ui/app/src/assets/uiConfigDefaults.ts]: We're using rounded version on the Launcher. Let's keep it consistent.

File: ui/app/src/assets/uiConfigDefaults.ts
Comment (rart) [inline ui/app/src/assets/uiConfigDefaults.ts]: This was mostly for testing purposes. Let's just leave `icon: { DescriptionOutlined }`

File: ui/app/src/assets/uiConfigDefaults.ts
Comment (rart) [inline ui/app/src/assets/uiConfigDefaults.ts]: Same deal, just leave one. Use our custom "Component" icon.

File: ui/app/src/assets/uiConfigDefaults.ts
Comment (rart) [inline ui/app/src/assets/uiConfigDefaults.ts]: Same

File: ui/app/src/assets/uiConfigDefaults.ts
Comment (rart) [inline ui/app/src/assets/uiConfigDefaults.ts]: Same

File: ui/app/src/assets/uiConfigDefaults.ts
Comment (rart) [inline ui/app/src/assets/uiConfigDefaults.ts]: Same

File: ui/app/src/assets/uiConfigDefaults.ts
Comment (rart) [inline ui/app/src/assets/uiConfigDefaults.ts]: ?

File: ui/app/src/components/Launcher/Launcher.tsx
Comment (rart) [inline ui/app/src/components/Launcher/Launcher.tsx]: x empty line

File: ui/app/src/components/PathNavigator/PathNavigatorHeader.tsx
Comment (rart) [inline ui/app/src/components/PathNavigator/PathNavigatorHeader.tsx]: Please add className & style as direct props of SystemIcon too (and merge accordingly)

File: ui/app/src/modules/Preview/ToolsPanel.tsx
Comment (rart) [inline ui/app/src/modules/Preview/ToolsPanel.tsx]: 👆

**Review by rart — CHANGES_REQUESTED** [review]

**Review by rart — APPROVED** [review]

**Pull request merged**
